lib/merch_calendar/date_calculator.rb

```diff
@@ -98,6 +98,18 @@ def julian_to_merch(julian_month)
       end
     end
 
-
+    def merch_months_in(start_date, end_date)
+      merch_months = []
+      prev_date = start_date - 2
```

Comment:
2 days. That insures prev_date and date are not equal for the first 
iteration.

Joel Strait wrote:
>
> In lib/merch_calendar/date_calculator.rb
> <https://github.com/stitchfix/merch_calendar/pull/5#discussion_r33818466>:
>
> > @@ -98,6 +98,18 @@ def julian_to_merch(julian_month)
> > end
> > end
> >
> > -
> > + def merch_months_in(start_date, end_date)
> > + merch_months = []
> > + prev_date = start_date - 2
>
> What is the |- 2| for?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stitchfix/merch_calendar/pull/5/files#r33818466>.